lib/diagnostic.js

```diff
@@ -4,8 +4,11 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-const min = function min() {
-
+const min = function min(sum, sum2) {
+  return Math.min.apply(sum, sum2);
+    }
+  }
+  return min();
```

Comment:
you are return 2 different things here, and you have two `}` that are going to throw syntax errors and break everything